src/Forms/FormField.php

```diff
@@ -742,12 +742,6 @@ public function getAttributesHTML($attributes = null)
         foreach ($attributes as $name => $value) {
             if ($value === true) {
                 $value = $name;
-            } else {
-                if (is_scalar($value)) {
-                    $value = (string) $value;
-                } else {
-                    $value = json_encode($value);
```

Comment:
Shouldn't we be throwing an exception if we get a non-scalar value at this point?

Reply:
I guess we could. Though, initially this patch was thought of as a revert of a merged feature.